easybuild/easyblocks/r/r.py

```diff
@@ -49,6 +50,16 @@ class EB_R(ConfigureMake):
     or latest library version (in that order of preference)
     """
 
+    def __init__(self, *args, **kwargs):
+        """Constructor for R easyblock."""
+        super(EB_R, self).__init__(*args, **kwargs)
+
+        lib_subdirs = ['', os.path.join('R', 'lib')]
+        self.module_load_environment.LD_LIBRARY_PATH = [os.path.join(d, x) for d in SEARCH_PATH_LIB_DIRS
+                                                        for x in lib_subdirs]
+        self.module_load_environment.LIBRARY_PATH = [os.path.join(d, x) for d in SEARCH_PATH_LIB_DIRS
+                                                     for x in lib_subdirs]
```

Comment:
We can simplify a bit:
```suggestion
        r_lib_subdirs = [os.path.join(libdir, 'R', 'lib') for libdir in SEARCH_PATH_LIB_DIRS]
        self.module_load_environment.LD_LIBRARY_PATH.extend(r_lib_subdirs)
        self.module_load_environment.LIBRARY_PATH.extend(r_lib_subdirs)
```

Reply:
fixed in 0c52618b9